cli/tsc/mod.rs

```diff
@@ -1002,7 +1003,7 @@ mod tests {
   // to allow removing some of the ops before taking a snapshot.
   #[ignore]
   #[test]
-  fn test_compiler_snapshot() {
+  async fn test_compiler_snapshot() {
```

Comment:
Did something change requiring this?

Reply:
Just consistency. I've added `#[tokio::test]` to this and the other one.